utility/serverrcon.js

```diff
@@ -90,6 +91,35 @@ class ServerRcon {
     }
   }
 
+  /**
+   * 
+   * Checks if the server is up to date via a steam API call.
+   * @function 
```

Comment:
Please add `@returns` in the function docs as well, since this isn't a `void` function :)